### DISCUSSION TITLE
Verify email

### DISCUSSION THREAD

**Discussion by GoncalofmSilva** [pocketbase/pocketbase#4565] [Q&A]
I'm trying to do a filter so that I can verify if the email exists on the collection users.

When I send an unexisting email I receive a 200 ok response with the others records

What I can do to receive a response that can give me what I want

Code:
```
const resultList = await pb.collection('users').getList(1, 100, {
      filter:`email!="${email}"`,
    });`
````

**Comment by ganigeorgiev** [comment]
You can search for other user emails only if:
- you are authenticated as admin
- you are authenticated as the owner of the user record
- the user has marked their email as public (aka. `emailVisibility` is `true`)

> I'm trying to do a filter so that I can verify if the email exists on the collection users.

The `email` field in the auth collections has builtin unique index defined so you usually don't have to perform any extra checks. In case there is an existing user, `pb.collection("users").create()` will throw an error (to inspect the error data you can explore the available normalized error fields in [JS SDK - Errors handling](https://github.com/pocketbase/js-sdk?tab=readme-ov-file#error-handling)).
